jsons/parameters.json

```diff
@@ -5973,7 +5958,8 @@
         "key_translations": {
             "pw-stats_style_1": "feature_columns",
             "ursgal_style_1": "feature_columns",
-            "umap_style_1": "feature_columns"
+            "umap_style_1": "feature_columns",
```

Comment:
Does the sorting has to be performed manually? umap_style_1 should be before ursgal_style_1